ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,184 @@
+			return GetResponse(response);
+		}
+
+		static async Task SubscriberToList(string AccountName, int ListId, string SubscriberEmail)
+		{
+			Email content = new Email
+			{
+				email = SubscriberEmail
+			};
+			HttpResponseMessage response = await client.PostAsJsonAsync("accounts/" + AccountName + "/lists/" + ListId + "/subscribers", content);
+			GetResponse(response);
+		}
+
+		static async Task<int> CreateCampaign(string AccountName, string Name, string FromName, string FromEmail, string Subject, string PreHeader, string ReplyTo)
+		{
+			Campaign content = new Campaign
+			{
+				name = Name,
+				fromName = FromName,
+				fromEmail = FromEmail,
+				subject = Subject,
+				preheader = PreHeader,
+				replyTo = ReplyTo
+			};
+			HttpResponseMessage response = await client.PostAsJsonAsync("accounts/" + AccountName + "/campaigns", content);
+			return GetResponse(response);
+		}
+
+		static async Task ContentToCampaign(string AccountName, int CampaignId, string HtmlCode, string ApiKey)
+		{
+			client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("text/html"));
+			HttpContent content = new StringContent(HtmlCode);
+			HttpResponseMessage response = await client.PutAsync("accounts/" + AccountName + "/campaigns/" + CampaignId + "/content", content);
+			GetResponse(response);
+		}
+
+		static async Task ListToCampaign(string AccountName, int CampaignId, int ListId)
+		{
+			ListCampaign.RootObject content = new ListCampaign.RootObject
+			{
+				lists = new List<ListCampaign.Lista>
+					{
+						new ListCampaign.Lista {id = ListId}
+					}
+			};
+			HttpResponseMessage response = await client.PutAsJsonAsync("accounts/" + AccountName + "/campaigns/" + CampaignId + "/recipients", content);
+			GetResponse(response);
+		}
+
+		static async Task SendCampaign(string AccountName, int CampaignId)
+		{
+			Campaign content = new Campaign
+			{
+				type = "immediate"
+			};
+			HttpResponseMessage response = await client.PostAsJsonAsync("accounts/" + AccountName + "/campaigns/" + CampaignId + "/shippings", content);
+			string ResponseBody = response.Content.ReadAsStringAsync().Result;
+			var Json = JsonConvert.DeserializeObject<CampaignToSend>(ResponseBody);
+			Console.WriteLine(response);
+			Console.WriteLine("Message :" + Json.message);
+		}
+
+		static void Main()
+		{
+			RunAsync().GetAwaiter().GetResult();
+		}
+
+		static async Task RunAsync()
+		{
+			string AccountName = "Nombre de la cuenta";
+			string ApiKey = "Api Key";
+
+			client.BaseAddress = new Uri("https://restapi.fromdoppler.com/");
+			client.DefaultRequestHeaders.Accept.Clear();
+			client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+			client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Token", ApiKey);
+
+			string ListName = "Lista creada por api";
+			string email = "Email de subscriptor a asociar";
+			string HtmlCode = "Codigo Html del contenido de la campaña";

```

Comment:
Maybe you could use English texts in place of Spanish ones.

Also, you could use more realistic examples, for example `accountname@email.com` for account name or `Hello <strong>World!!!</strong>` for HtmlCode.

Reply:
Thanks for the review @andresmoschini .
 I will make the changes you suggested.